deepspeed/utils/mappings.py

```diff
@@ -0,0 +1,72 @@
+import torch
+import deepspeed
+
+
+def _gather_tokens(input_, dim=0):
+    """Gather tensors and concatinate them along a dimension"""
```

Comment:
concatenate? Better to do a spelling check.

Reply:
yep, the comments can be all over the place. I'll fix them.